google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/sdk/ManagedCloudSdkService.java

```diff
@@ -123,9 +102,9 @@ public void removeStatusUpdateListener(SdkStatusUpdateListener listener) {
     statusUpdateListeners.remove(listener);
   }
 
-  public void cancelInstall() {
-    if (runningInstallationJob != null) {
-      runningInstallationJob.cancel(true);
+  public void cancelInstallOrUpdate() {
```

Comment:
this will get called by code in your next PR when you implement progress bars, correct?

Reply:
yes, exactly